umapi_client/functional.py

```diff
@@ -308,6 +308,32 @@ def delete_account(self):
         self.append(removeFromDomain={})
         return None
 
+    def split_groups(self, index, verb, group_type, max_groups):
+        """
+        Split a long list of groups in an add/remove command
+        :param index: index of command to split
+        :param verb: add or remove
+        :param group_type: group type - product, productConfiguration or usergroup
+        :param max_groups: max size of groups (if group list is larger than max it will be split)
+        :return:
+        """
+        if index > len(self.commands):
+            raise ArgumentError(six.text_type("Index {} not found in commands list").format(index))
+        if verb not in self.commands[index]:
+            raise ArgumentError(six.text_type("'{}' not specified in command").format(verb))
+        if group_type not in self.commands[index][verb]:
+            raise ArgumentError(six.text_type("'{}' not specified in command").format(group_type))
+        groups = self.commands[index][verb][group_type]
+        if len(groups) > max_groups:
```

Comment:
this code is a little tortured.  see comments in the connection file about a cleaner way to do this.